ChineseFilter/sections/specific.txt

```diff
@@ -21,6 +21,7 @@ $script,xmlhttprequest,third-party,domain=00ksw.com|01zww.com|0biquge.com|114zw.
 iwatchme2u.com##.video-ad-box
 ||r.trwl1.com/s1/$subdocument
 ggjav.com###ads_6
+pixnet.net##.pixnet-ad
```

Comment:
Can you show the element with opened devtools in the screenshot?
I've manually added a rule blocking empty space because your rule isn't enough.